client/components/PostQuestionModal.jsx

```diff
@@ -123,30 +149,33 @@ const PostButton = styled(Button)`
 `;
 
 
-const PostQuestionModal = ({show}) => {
+const PostQuestionModal = ({ show, toggleModal }) => {
   return (
     (!show
```

Comment:
Maybe move " !show ? null :"  to one line for better. readability

Reply:
Will make this refactor, thank you